app-tests/quantumespresso/quantumespresso_check.py

```diff
@@ -0,0 +1,45 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class quantumespresso_check(rfm.RunOnlyRegressionTest):
+    modules = ['QuantumESPRESSO']
+    executable = 'pw.x'
+    maintainers = ['mszpindler']
+    strict_check = True
```

Comment:
I think this is the default. No need to set it.

Reply:
You mean `strict_check` being on by default?

Reply:
Yes. I think it is, right?

Reply:
You are right, I am removing these lines.